### ISSUE TITLE
Feature request: separate system log notes and OdooBot messages in the chatter.

### ISSUE THREAD

**Issue by obayit** [odoo/odoo#73391]
Current behavior:
All messages appear in one place.
Expected behavior:
System messages (e.g: field changes) and OdooBot messages should be in a separate section/tab.

**Comment by pedrobaeza** [comment]
I don't think this can be even considered for v15, that is barely at freeze time, but the wishlist is registered for any version.

**Comment by cunivca** [comment]
Hello, this is a request we get from clients frequently. 
Any information on whether this will be added in the new versions? 

Thanks!

**Comment by Talebyassine8** [comment]
Hi @cunivca,

We have developed an odoo that i think responds to this issue, it is available in the app store:
https://apps.odoo.com/apps/modules/17.0/teos_chatter_tabs

**Comment by seb-odoo** [comment]
We now have search in messages to help with filtering those, this should cover most use cases.

There's currently no plan to create more separation or tabs.
